test/rubygems/test_gem_ext_rake_builder.rb

```diff
@@ -39,6 +39,36 @@ def test_class_build
     end
   end
 
+  # https://github.com/rubygems/rubygems/pull/1819
+  #
+  # It should not fail with a non-empty args list either
+  def test_class_build_with_args
+    File.open File.join(@ext, 'mkrf_conf.rb'), 'w' do |mkrf_conf|
```

Comment:
There's a fair amount of duplication between this test-case and the one above. A meaningful one would be to extract would be this file creation. Especially since the two files are identical.